static/js/tools/scatter/chart.tsx

```diff
@@ -200,9 +202,15 @@ function Chart(props: ChartPropsType): JSX.Element {
       <Row>
         <Card id="no-padding">
           <div className="chart-title">
-            <h3>{props.yLabel}</h3>
+            <h3>
+              {props.yLabel}
+              {props.yDate && ` (${props.yDate})`}
```

Comment:
if it's the requested date, we might not be able to use it directly in the title, and it might be better to use the dates from data like we do elsewhere.